Jenkinsfile

```diff
@@ -43,6 +43,14 @@ pipeline {
       }
     }
 
+    stage('Validate') {
```

Comment:
This should probably be one of the first stages to run. Right now it seems like the build is failing for other reasons and so you're unable to tell if validation is succeeding or not.